demo/src/FlatParametersTab.jsx

```diff
@@ -198,7 +198,7 @@ const EXAMPLE_PARAMETERS = [
     },
 ];
 
-export const FlatParametersTab = () => {
+function FlatParametersTab() {
```

Comment:
function? In TS that would be
```typescript
const FlatParametersTab: FunctionComponent = () => {
```

Reply:
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/function-component-definition.md